tests/test_amplitude.py

```diff
@@ -35,10 +36,12 @@ def test_init_pass_args():
         api_key=api_key,
         include_user_data=(not settings.INCLUDE_USER_DATA),
         include_group_data=(not settings.INCLUDE_GROUP_DATA),
+        min_id_length=settings.MIN_ID_LENGTH + 1,
```

Comment:
As `MIN_ID_LENGTH` should be removed from the `test/settings.py` please set this to it's own variable instead of using `settings.MIN_ID_LENGTH`